creator-node/test/monitorState.jobProcessor.test.js

```diff
@@ -408,15 +409,15 @@ describe('test monitorState job processor', function () {
     )
     expect(getUnhealthyPeersStub).to.not.have.been.called
     expect(buildReplicaSetNodesToUserWalletsMapStub).to.not.have.been.called
-    expect(retrieveClockStatusesForUsersAcrossReplicaSetStub).to.not.have.been
+    expect(retrieveUserInfoFromReplicaSetStub).to.not.have.been
       .called
     expect(computeUserSecondarySyncSuccessRatesMapStub).to.not.have.been.called
     verifyJobResult({
       jobResult,
       lastProcessedUserId: LAST_PROCESSED_USER_ID,
       users: [{ user_id: LAST_PROCESSED_USER_ID }],
       unhealthyPeers: new Set(),
-      replicaSetNodesToUserClockStatusesMap: {},
+      replicaToUserInfoMap: {},
       userSecondarySyncMetricsMap: {}
     })
   })
```

Comment:
it's great to see all the tests updated -- that looks like it was a lot of work 😅